src/CommercetoolsAuth0.ts

```diff
@@ -222,6 +226,37 @@ export class CommercetoolsAuth0 {
     return null
   }
 
+  /**
+   * Return the customer details from the Commercetools graphQl response
+   *
+   * @param id - Commercetools customer id
+   */
+  public async getCustomerDetails(customerId: string): Promise<CustomerDetails> {
+    const customerDetails: CustomerDetails = {
+      id: customerId,
+      firstName: '',
+      lastName: '',
+      customerGroupKey: '',
+      newUserCreated: false,
+    }
+    try {
+      const response = await this.client.graphql({
+        data: {
+          query: `{ customer(id: "${customerId}"){firstName
+          lastName
+          customerGroup {
+          key }}}`,
+        },
+      })
+      customerDetails.firstName = response?.data?.customer?.firstName || ''
+      customerDetails.lastName = response?.data?.customer?.lastName || ''
+      customerDetails.customerGroupKey = response?.data?.customer?.customerGroup?.key || ''
```

Comment:
You could just use:
```
return {
  id: customerId,
  firstName: response?.data?.customer?.firstName || '',
  lastName: response?.data?.customer?.lastName || '',
  groupKey: response?.data?.customer?.customerGroup?.key || '',
}
```
...and change the return type to `Promise<Omit<CustomerDetails, 'newUserCreated'>>`